### PR TITLE
Update global post scripts for coupled

### PR DESCRIPTION
**Pull request by WalterKolczynski-NOAA**
Updates global post to handle extra atm output for coupled runs (post for other components is still handled separately). This mirrors the same change to global-workflow which also brings these two scripts under g-w control (see NOAA-EMC/global-workflow/pull/766)

### CODE REVIEW

File: scripts/exgfs_atmos_nceppost.sh [hunk 1]
```diff
@@ -39,11 +39,12 @@ set -x
 cd $DATA
 
 # specify model output format type: 4 for nemsio, 3 for sigio
-msg="HAS BEGUN on `hostname`"
+msg="HAS BEGUN on $(hostname)"
 postmsg "$msg"
 
 export POSTGPSH=${POSTGPSH:-$USHgfs/gfs_nceppost.sh}
 export GFSDOWNSH=${GFSDOWNSH:-$USHgfs/fv3gfs_downstream_nems.sh}
+export GFSDOWNSHF=${GFSDOWNSHF:-$USHgfs/inter_flux.sh}
```
Comment (WenMeng-NOAA) [inline scripts/exgfs_atmos_nceppost.sh:47]: @WalterKolczynski-NOAA I saw you add some clean up in this script. I would like to add the capability of  of GFSDONSHF only before the all GFS post processing scripts transition to global-workflow repository. We have been working on a couple of UPP major upgrades before July and have no additional resources. Thanks!

Comment (WalterKolczynski-NOAA) [reply scripts/exgfs_atmos_nceppost.sh:47]: Okay, let me start over then.

File: ush/fv3gfs_downstream_nems.sh [hunk 2]
```diff
@@ -158,193 +157,136 @@ while [ $nset -le $totalset ]; do
...
-     if [ "$PGBS" = "YES" ]; then
-       cat pgb2file_${fhr3}_${iproc}_0p5  >> pgb2file_${fhr3}_0p5
-       cat pgb2file_${fhr3}_${iproc}_1p0  >> pgb2file_${fhr3}_1p0
-       if [ "$PGB1F" = 'YES' ]; then
-         cat pgbfile_${fhr3}_${iproc}_1p0   >> pgbfile_${fhr3}_1p0
-       fi
-     fi
+      cat pgb2file_${fhr3}_${iproc}_0p25 >> pgb2file_${fhr3}_0p25
+      if [ "$PGBS" = "YES" ]; then
+        # cat pgb2file_${fhr3}_${iproc}_0p5  >> pgb2file_${fhr3}_0p5
```
Comment (WenMeng-NOAA) [inline ush/fv3gfs_downstream_nems.sh:196]: @WalterKolczynski-NOAA This is the operational GFS key products. You might add a switch to turn off for developer experiment.

File: ush/fv3gfs_downstream_nems.sh [hunk 3]
```diff
@@ -93,13 +92,13 @@ $WGRIB2 $PGBOUT2 | grep -F -f $paramlist | $WGRIB2 -i -grib  tmpfile1_$fhr3 $PGB
...
-   $WGRIB2 $PGBOUT2 | grep -F -f $paramlistb | $WGRIB2 -i -grib  tmpfile2_$fhr3 $PGBOUT2
-   export err=$?; err_chk
+  $WGRIB2 $PGBOUT2 | grep -F -f $paramlistb | $WGRIB2 -i -grib  tmpfile2_$fhr3 $PGBOUT2
+  export err=$?; err_chk
 fi
 
 #-----------------------------------------------------
 #-----------------------------------------------------
-if [ $machine = WCOSS -o $machine = WCOSS_C -o $machine = WCOSS_DELL_P3 -o $machine = HERA -o $machine = ORION -o $machine = JET -o $machine = S4 -o $machine = WCOSS2 ]; then
+#if [ $machine = WCOSS -o $machine = WCOSS_C -o $machine = WCOSS_DELL_P3 ]; then
```
Comment (WenMeng-NOAA) [inline ush/fv3gfs_downstream_nems.sh:101]: @WalterKolczynski-NOAA We would like to keep this capability to support the UPP users running gfs post processing workflow on platforms without MPMD before GFS post processing workflow transition is completed.

Comment (WalterKolczynski-NOAA) [reply ush/fv3gfs_downstream_nems.sh:101]: I'm looking to have that transition happen quickly. I already opened a [draft PR open](https://github.com/NOAA-EMC/global-workflow/pull/771) in g-w that copies the rest over.

File: ush/fv3gfs_downstream_nems.sh (hunk 2 above)
Comment (WalterKolczynski-NOAA) [reply ush/fv3gfs_downstream_nems.sh:196]: I had restored them all in the other script, I just missed these. I'm putting them back in for the g-w PR.

**Pull request closed without merging**